### ISSUE TITLE
2.1.0 release

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#726]
**Ready** when all other issues on this milestone are **Done** and closed.

- [x] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
- [x] if necessary, change `VERSION_SUFFIX` on [CI Server](http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929)
      to appropriate "-beta123" or "" (for non-betas) value and initiate a build
- [x] check build
-  edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
    - [x] complete release notes, mentioning non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
    - [x] attach nupkg
    - [x] publish the release
- [x] push NuGet package
- [x] de-list pre-release or superseded buggy NuGet packages if present
- [x] update website with contributors list (if in place)
- [x] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [x] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [x] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
- [x] use `rake next_version[new_version]` to
    - create a new branch
    - change CommonAssemblyInfo.cs to expected minor version (of form _xx.yy.zz_)
    - push to origin
    - create PR to upstream master
    - create a new milestone for the next release
    - create new issue (like this one) for the next release, adding it to the new milestone
    - create a new draft GitHub Release
- [x] close this milestone

**Comment by blairconrad** [comment]
The description doesn't match the new way we intend to work, but after #722 and #665 are merged/fixed, I'll update it to match.

**Comment by blairconrad** [comment]
Now that #782 has been merged, shall we consider a release? It's been _20 days_ already! Or would we like #755 and #758 in here as well?

**Comment by adamralph** [comment]
I think we can go ahead now. I'm all for continuous release.

**Comment by adamralph** [comment]
Mind you, #755 is *extremely* closely related. Might be neater to get that in too.

**Comment by adamralph** [comment]
btw - what's #782?

**Comment by adamralph** [comment]
Presumably you mean https://github.com/FakeItEasy/FakeItEasy/issues/482

**Comment by blairconrad** [comment]
Err, yes. I did say that it was early in Canada. Ahem.
Thanks!

**Comment by blairconrad** [comment]
Sounds good to me, @adamralph. Let's put #755 in.

**Comment by blairconrad** [comment]
Unreadying this issue until #755 is done.

**Comment by adamralph** [comment]
Ready to go again!

**Comment by thomaslevesque** [comment]
I updated the release notes with newly fixed issues and contributor names

**Comment by adamralph** [comment]
Not ready again :smile:

**Comment by blairconrad** [comment]
… and **ready**. Shall we release while we have a chance?

**Comment by adamralph** [comment]
:thumbsup: :shipit:

**Comment by adamralph** [comment]
https://twitter.com/fakeiteasyfx/status/744867154143621120

**Comment by blairconrad** [comment]
:blush:
